client/app/telecom/telephony/line/phone/mgcpIpRestriction/telecom-telephony-line-phone-mgcpIpRestriction.html

```diff
@@ -108,10 +109,11 @@ <h2 data-translate="loading"></h2>
                                 data-ng-disabled="mgcpDefaultIpRestrictionForm.$invalid ||
                                                   !MgcpIpRestrictionCtrl.hasMgcpDefaultIpRestrictionChanged() ||
                                                   MgcpIpRestrictionCtrl.isChangingMgcpDefaultIpRestriction">
+                                <bouncing-box-loader class="d-inline-block align-middle mr-2"
```

Comment:
Are these classes necessary here with the new `bouncing-box-loader` tag?

Reply:
Yes they are! By default it's computed as a block element.